<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fixForm</class>
 <widget class="QDialog" name="fixForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FIX</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="filenameLineEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
     <item>
      <widget class="QPlainTextEdit" name="infoTextEdit"/>
     </item>
     <item>
      <widget class="imageViewer" name="previewWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>120</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>120</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="drmSegmentsView" name="label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fixPushButton">
       <property name="text">
        <string>Fix</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>imageViewer</class>
   <extends>QWidget</extends>
   <header>widgets/imageviewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>drmSegmentsView</class>
   <extends>QLabel</extends>
   <header>drmsegmentsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>fixForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fixPushButton</sender>
   <signal>clicked()</signal>
   <receiver>fixForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
